stress/src/metrics_counter.rs

```diff
@@ -4,6 +4,9 @@
     Hardware: Intel(R) Xeon(R) Platinum 8370C CPU @ 2.80GHz, 16vCPUs,
     RAM: 64.0 GB
     ~9.5 M /sec
+
+    Hardware: AMD EPYC 7763 64-Core Processor - 2.44 GHz, 16vCPUs,
```

Comment:
You probably intended to replace the previous entry instead of appending a new one.

Reply:
keeping both :D as we have AMD and Intel machines, so no harm keeping both :)

Reply:
Okay 😀 We should update the stress test results for Intel then. It should have gone up from 9.5 M/sec.